File: src/sbt-test/lagom-bundle-plugin/multi-services-one-project-same-service-name-java/payment-impl/src/main/java/impl/CreditServiceImpl.java
package impl;

import akka.NotUsed;
import com.lightbend.lagom.javadsl.api.ServiceCall;
import java.util.concurrent.CompletableFuture;
import javax.inject.Inject;
import api.CreditService;

import akka.stream.javadsl.Source;

public class CreditServiceImpl implements CreditService {

  @Override
  public ServiceCall<NotUsed, NotUsed> credit() {
    return request -> CompletableFuture.completedFuture(NotUsed.getInstance());
  }
}

File: src/sbt-test/lagom-bundle-plugin/multi-services-one-project-same-service-name-scala/payment-impl/src/main/scala/impl/CreditServiceImpl.scala
package impl

import akka.NotUsed
import api.CreditService
import scala.concurrent.Future
import com.lightbend.lagom.scaladsl.api.ServiceCall

class CreditServiceImpl extends CreditService {
  override def credit = ServiceCall { x =>
    Future.successful(NotUsed)
  }
}